### DISCUSSION TITLE
Angular Testing Library with Vitest

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#354] [Announcements]
# Angular Testing Library with Vitest

Because of the Vitest plugin provided by Analog it just takes a few steps and minutes to test your Angular components with Angular Testing Library.

https://timdeschryver.dev/blog/angular-testing-library-with-vitest

**Comment by timsar2** [comment]
This is why this blog is set in my favorite list ♥️

**Comment by Narbe66** [comment]
Hi,

I was wondering if createMock is working with vitest without jest. Is there an equivalent to createMock for vitest?

**Comment by anthony-bernardo** [comment]
Hi

I'm getting Need to call TestBed.initTestEnvironment() first strangly
